cypress/integration/unit_tests/add_disaster_test.js

```diff
@@ -261,55 +267,53 @@ describe('Unit tests for add_disaster page', () => {
 
   it('tests list cell', () => {
     const chocolate = 'chocolate';
```

Comment:
nit about these tests' style. It doesn't harm anything, but I also don't see the advantage of using variables for strings that are the same as the variables. You're just saving the single quotes? Especially below where you're joining the strings it might actually look neater to have them as literals.

Reply:
Isn't the advantage that you can't accidentally change one usage of the var without changing others? I thought generally good practice to variablize reused strings.

Reply:
I'm less convinced of it in tests. Not a big deal either way